types/schemas/prototypeApplication/enums/PropertyTypes.ts

```diff
@@ -2729,6 +2729,7 @@ export type PropertyType =
   | CommercialUtilityWaterTreatmentWater
   | CommercialUtilityWaterWaste
   | DualUse
+  // eslint-disable-next-line @typescript-eslint/ban-types
```

Comment:
ESLint was complaining about (I think) the keyword 'object' even though `Object` is defined as its own custom type further up the file.